expression/builtin.go

```diff
@@ -147,6 +147,7 @@ var Funcs = map[string]Func{
 	ast.Abs:     {builtinAbs, 1, 1},
 	ast.Ceil:    {builtinCeil, 1, 1},
 	ast.Ceiling: {builtinCeil, 1, 1},
+	ast.Floor:   {builtinFloor, 1, 1},
```

Comment:
also add this for this line: https://github.com/pingcap/tidb/pull/2484/files#diff-cdbd511e3e5f2bbe0a3c5c173d3938c2L298